src/screens/ChatScreen/AnonymousChatScreen.tsx

```diff
@@ -135,7 +135,12 @@ const AnonymousChatScreen = () => {
       />
       <View style={styles.inputContainer}>
         <InputMessageV2
-          onSendButtonClicked={sendChatMutation.mutate}
+          onSendButtonClicked={(message: string, attachments: any) => {
+            sendChatMutation.mutate({
+              message,
+              attachments
+            } as any);
+          }}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here improve the readability and maintainability of the code by explicitly passing `message` and `attachments` as separate parameters to the `sendChatMutation.mutate` function. However, it's important to ensure that the types of `message` and `attachments` are correctly handled in the `sendChatMutation.mutate` function. If not, this could lead to potential type errors or unexpected behavior.

Also, you're casting the object `{ message, attachments }` as `any`. This is generally not a good practice because it bypasses TypeScript's static type checking, which can lead to runtime errors that are hard to debug. It would be better to define an interface for the expected shape of the object and use that instead of `any`.

```diff
-             sendChatMutation.mutate({
-               message,
-               attachments
-             } as any);
+             const chatData: ChatData = { message, attachments };
+             sendChatMutation.mutate(chatData);
```
In the above snippet, `ChatData` is a hypothetical interface that you should define according to your requirements.


<!-- This is an auto-generated comment by OSS CodeRabbit -->